shared/src/Datadog.Trace.ClrProfiler.Native/single_step_guard_rails.cpp

```diff
@@ -171,17 +181,19 @@ bool SingleStepGuardRails::ShouldForceInstrumentationOverride(const std::string&
             "SingleStepGuardRails::ShouldForceInstrumentationOverride: ",
             EnvironmentVariables::ForceEolInstrumentation,
             " enabled, allowing unsupported runtimes and continuing");
+        SetInjectResult("success", "Force instrumentation enabled", "success_forced");
```

Comment:
Should we mention in the message that you're forcing instrumentation with an incompatible runtime? Or is it enough that it's implied?